yb-voyager/src/srcdb/mysql.go

```diff
@@ -46,6 +47,8 @@ func newMySQL(s *Source) *MySQL {
 
 func (ms *MySQL) Connect() error {
 	db, err := sql.Open("mysql", ms.getConnectionUri())
+	db.SetMaxOpenConns(1)
+	db.SetConnMaxIdleTime(5 * time.Minute)
```

Comment:
So what will be the behaviour? In case conn is idle for more than 5 minutes then it will close it and let's the code needs a connection after 6 or 7 minutes, will the pool make a new connection?

Reply:
Yes, the pool will automatically create a new connection if required